src/Controller/BlogController.php

```diff
@@ -75,12 +75,17 @@ public function index(Request $request, int $page, string $_format, PostReposito
     #[Route('/posts/{slug}', methods: ['GET'], name: 'blog_post')]
     public function postShow(Post $post): Response
     {
-        // Symfony's 'dump()' function is an improved version of PHP's 'var_dump()' but
+        // Symfony's 'dump()' and 'dd()' functions are an improved version of PHP's 'var_dump()' but
```

Comment:
``dd` is not just an improved `var_dump`. It is dump + die, which is quite different that just dumping.

Reply:
you are right, now the phrasing is weird
do you think I revert and adding the `dd()` is not worth it? or keep it but rephrase?
thank you @stof